docs/platforms/javascript/common/session-replay/troubleshooting.mdx

```diff
@@ -17,6 +17,10 @@ This guide aims to extend the <PlatformLink to="/troubleshooting/">main troubles
 
 Canvas is supported in SDK versions >= `7.98.0`. Please see the <PlatformLink to="/session-replay/#canvas-recording">canvas setup documention</PlatformLink> to get started with canvas recordings.
 
+## My `canvas` has been tainted
```

Comment:
I don't think users will know what "tainted" means in this context - I think we should move this paragraph into the above section because it will look like canvas is not being captured.

We should say something like, If you are on a supported SDK version and your canvas elements still aren't being captured, check if you have images or videos inside of your canvas that are loaded from foreign origins. You may also see a `SecurityError` being thrown when trying to use the replay canvas integration. ...